### ISSUE TITLE
Announcement

### ISSUE THREAD

**Issue by razzeee** [GrayJack/language-elm-treesitter#1]
Hey,

I felt like asking, since you are doing releases :)

Will you be announcing this officially on slack, discourse and reddit at some point in time?

Cheers

**Comment by GrayJack** [comment]
Yes, I plan on discourse and reddit, I don't have slack, so that one will be missing.

But for now, I'm just getting some code in the atom so see if it gets some things wrong

**Comment by razzeee** [comment]
Nice, I'll probably spot that anyway, can cross post it to slack, if you want me to.
